pkg/executor/test/simpletest/simple_test.go

```diff
@@ -818,3 +818,44 @@ func TestKillStmt(t *testing.T) {
 	tk.MustExecToErr("kill rand()", "Invalid operation. Please use 'KILL TIDB [CONNECTION | QUERY] [connectionID | CONNECTION_ID()]' instead")
 	// remote kill is tested in `tests/globalkilltest`
 }
+
+func TestSelectWhereInvalidDSTTime(t *testing.T) {
+	store := testkit.CreateMockStore(t)
+	tk := testkit.NewTestKit(t, store)
+	tk.MustExec("use test")
+	tk.MustExec("create table t (id int, ts timestamp)")
```

Comment:
I assume `timestamp` and `timestamp(6)` are expected to behave the same?

Reply:
Yes. I just tested manually and it is the same, but would prefer to not include `timestamp(6)` in this test. I am currently working on a more covering tests for timestamp, which also includes timestamp with fractions, which should cover this as well.